lms/djangoapps/ccx/views.py

```diff
@@ -59,6 +61,18 @@
 TODAY = datetime.datetime.today  # for patching in tests
 
 
+class CCXCourseFullException(Exception):
+    """
+    Custom Exception for full ccx courses
+    """
+
```

Comment:
Generally we explicitly add in a `pass`. see openedx/core/djangoapps/content/course_structures/api/v0/errors.py